src/bot.coffee

```diff
@@ -3,6 +3,8 @@
 SlackClient                                                                       = require "./client"
 pkg                                                                               = require "../package"
 
+Events      = require("@slack/client").CLIENT_EVENTS
```

Comment:
small nitpick: I think this will get the spacing aligned with the rest of the requires above
```suggestion
Events                                                                            = require("@slack/client").CLIENT_EVENTS
```